### DISCUSSION TITLE
ESP8266 reset in function GSheet.values.append

### DISCUSSION THREAD

**Discussion by diegostf** [mobizt/ESP-Google-Sheet-Client#5] [Q&A] [answered]
I'm testing the Append.ino example. On ESP32 it works great, but on ESP8266, reset on the line: 
bool success = GSheet.values.append(&response /* returned response */, "<spreadsheetId>" /* spreadsheet Id to append */, "Sheet1!B3" /* range to append */, &valueRange /* data range to append */);

Error: 
----------------------------
Exception (3):
epc1=0x4000bf64 epc2=0x00000000 epc3=0x00000000 excvaddr=0x4026a719 depc=0x00000000

>>>stack>>>

ctx: cont
sp: 3ffffbb0 end: 3fffffc0 offset: 0190
3ffffd40:  00000001 3fff0064 3ffffd60 402114e9  
3ffffd50:  00000001 3fff0064 3ffffdf0 40206852  
3ffffd60:  00000000 0016001f 00000000 3fff0cf4  
3ffffd70:  00000018 00000000 3ffffdf0 4020b875  
3ffffd80:  00000001 4026a719 3ffe89a6 40100927  
3ffffd90:  00000000 00000000 00000000 3ffe897b  
3ffffda0:  3ffe8630 3ffeeee0 3ffeef7c 40206a1d  
3ffffdb0:  3ffffdf0 3ffe8999 3ffe8e84 3ffe8e84  
3ffffdc0:  3ffe8e84 3ffe8e84 00000023 3ffe897b  
3ffffdd0:  3ffe8630 3ffeeee0 3ffeef7c 40204d71  
3ffffde0:  3ffe8e84 3ffe8e84 3ffe8e84 3ffe8e84  
3ffffdf0:  4021f234 3ffe0509 00000000 ffffffff  
3ffffe00:  ffffffff ffffffff ffffffff 00000000  
3ffffe10:  00000000 00000000 00000000 00000000  
3ffffe20:  00000000 00000000 00000000 00000000  
3ffffe30:  00000000 00000000 ffffffff 00000000  
3ffffe40:  ffffffff 00000000 00000000 00000000  
3ffffe50:  00000000 00000000 00000000 00000000  
3ffffe60:  00000000 00000000 00000000 3fff0c7c  
3ffffe70:  00000000 00000000 00000000 4021f234  
3ffffe80:  00000509 00000000 ffffffff ffffffff  
3ffffe90:  ffffffff ffffffff 00000000 00000000  
3ffffea0:  00000000 00000000 00000000 00000000  
3ffffeb0:  00000000 00000000 00000000 00000000  
3ffffec0:  00000000 ffffffff 00000000 ffffffff  
3ffffed0:  3fff0000 00000000 00000000 00000000  
3ffffee0:  00000000 00000000 00000000 00000000  
3ffffef0:  00000000 00000000 00000000 00000000  
3fffff00:  00000000 00000000 3fff0064 00000001  
3fffff10:  3ffeefc4 0000002f 3fff0974 40100b06  
3fffff20:  3fff07d4 3ffeefc0 3fff0960 40229400  
3fffff30:  00000000 00000000 00000001 40100320  
3fffff40:  4026a745 3fff009c 00000001 3ffeefc0  
3fffff50:  3ffeefc4 00000001 40210f08 3fffefa0  
3fffff60:  007a1200 361afa78 3fff0000 40206306  
3fffff70:  3fff095c 3fff095c 00000020 3ffef028  
3fffff80:  00000000 00000000 00000001 40100320  
3fffff90:  3fffdad0 00000000 3ffef014 3ffef028  
3fffffa0:  3fffdad0 00000000 3ffef014 40211c50  
3fffffb0:  feefeffe feefeffe 3ffe8660 40100dc1  
<<<stack<<<

--------------- CUT HERE FOR EXCEPTION DECODER ---------------

 ets Jan  8 2013,rst cause:2, boot mode:(3,6)

load 0x4010f000, len 3460, room 16 
tail 4
chksum 0xcc
load 0x3fff20b8, len 40, room 4 
tail 4
chksum 0xc9
csum 0xc9
v00079bd0
~ld

No code modifications. Only added header items.

Can you help me?

**Comment by mobizt** [comment, marked as answer]
You should read [this](https://github.com/mobizt/ESP-Google-Sheet-Client#ide-configuaration-for-esp8266-mmu---adjust-the-ratio-of-icache-to-iram).

**Reply by diegostf** [reply]
Great. Tks a lot.
